ci/travis/format.sh

```diff
@@ -132,6 +142,19 @@ mypy_on_each() {
     popd
 }
 
+format_java_files() {
+  # read files from pipe
+  while read -r file; do
+      printf "%s\n" "$file" >> /tmp/ray_java_files_to_format
+  done
+  if command -v java >/dev/null & [ -f "$GOOGLE_JAVA_FORMAT_JAR" ]; then
+    printf "Start format java files:\n%s" "$(cat /tmp/ray_java_files_to_format)"
```

Comment:
This is too much. We should either print only changed files or not at all. For your reference, the current `format.sh` only prints changed C++ files.